public/app/features/plugins/sandbox/sandbox_plugin_loader.ts

```diff
@@ -33,7 +31,7 @@ const pluginLogCache: Record<string, boolean> = {};
 export async function importPluginModuleInSandbox({ pluginId }: { pluginId: string }): Promise<System.Module> {
   patchWebAPIs();
   try {
-    const pluginMeta = await getPluginSettings(pluginId);
+    const pluginMeta = await getPluginLoadData(pluginId);
```

Comment:
Following [Jack's comment](https://github.com/grafana/grafana/pull/86702/files#r1576275937) we can probably remove the `await` here, too?

Reply:
Indeed. I was trying a different approach and didn't remove async. Thanks for catching it.